src/xrScriptEngine/script_engine.cpp

```diff
@@ -524,7 +524,7 @@ struct raii_guard : private Noncopyable
 #ifdef DEBUG
             static const bool break_on_assert = !!strstr(Core.Params, "-break_on_assert");
 #else
-            static const bool break_on_assert = true;
+            static const bool break_on_assert = false;	// xxx: true will hide real lua error, LuaJIT problem?
```

Comment:
Original look of log doesn't suppose printing of luabind errors. (since they disabled in original game) And even if we find a way to print luabind error in raii_guard::~raii_guard(), its errors doesn't contain info about lua-file name\line, since it generated only by LuaJIT. (and luaL_traceback can be accessed only inside pcall callback)